server.ts

```diff
@@ -1,5 +1,5 @@
 import 'source-map-support/register';
-import env from './env';
+const env = require('./env');
```

Comment:
if you need to import `js` files, please consider `allowJs` [compiler option](https://www.typescriptlang.org/docs/handbook/compiler-options.html)